fluentd/configs.d/openshift/filter-post-events.conf

```diff
@@ -0,0 +1,9 @@
+<filter kubernetes.**>
+  @type record_transformer
+  enable_ruby true
+  auto_typecast true
+  <record>
+    event ${ if record.key?("event") then record["kubernetes"]["event"]=record["event"]; "" else record["event"] end }
```

Comment:
I suggest something like this
```
  unused1 ${ if record.key?("event") then record["kubernetes"]["event"]=record.delete("event"); ""}
  unused2 ${ if record.key?("verb") then record["kubernetes"]["event"]["verb"]=record.delete("verb"); ""}
</record>
remove_keys unused1,unused2
```
Using `delete` will "move" the contents.  Using the `unused` fields you can just use any arbitrary ruby code you want to change the structure of the `record`.